xml/System.Windows.Data/BindingOperations.xml

```diff
@@ -190,6 +190,65 @@
         </remarks>
       </Docs>
     </Member>
+
+    <MemberGroup MemberName="EnableCollectionSynchronization">
```

Comment:
interesting. didn't know we had/could add things like this manually

Reply:
@mairaw, `<MemberGroup>` was created during migration if an overload list topic had some content other than the summary/description. But otherwise, it's not present. That also means that, when overload list topics had only a summary but no other content, that description wasn't migrated; for example, see [String.CompareOrdinal](https://docs.microsoft.com/en-us/dotnet/api/system.string.compareordinal?view=netcore-2.0).

Reply:
Humm, we should open a bug then to migrate those.